src/hearings/converters/reasonable-adjustments.amended.converter.ts

```diff
@@ -0,0 +1,28 @@
+import * as _ from 'lodash';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { State } from '../store/reducers';
+import { IsAmendedConverter } from './is-amended.converter';
+import { PartyDetailsModel } from '../models/partyDetails.model';
+import { PartyType } from '../models/hearings.enum';
+
+export class ReasonableAdjustmentsAmendedConverter implements IsAmendedConverter {
+  public transformIsAmended(hearingState$?: Observable<State>): Observable<boolean> {
+    return hearingState$.pipe(map((state) => {
+      return this.checkReasonableAdjustments(state.hearingRequest.hearingRequestMainModel.partyDetails,
+        state.hearingRequestToCompare.hearingRequestMainModel.partyDetails);
+    }));
+  }
+
+  public checkReasonableAdjustments(originalPartyDetails: PartyDetailsModel[], partyDetailsToCompare: PartyDetailsModel[]) {
+    return partyDetailsToCompare.some((party) => {
```

Comment:
I'm not sure that some() is appropriate here. some will look at the parties until it finds one that matches, and then it will stop. I think you probably want every() rather than some, and you want a test case with say 3 parties, where the first 2 match and the 3rd doesn't

Reply:
Hi Andy, the function checkReasonableAdjustments checks if the reasonable adjustments are different between the parties (it will return true for the first party it finds with different reasonable adjustments and stop processing at that point).